config.yaml

```diff
@@ -13,3 +13,8 @@ options:
 
       The value for this config must be a JSON array of credential objects, like this:
         [{"url": "https://my.registry:port", "username": "user", "password": "pass"}]
+  disable_gpu:
```

Comment:
Can we have this a string with something like "auto, cpu, gpu"?

Reply:
Would you ever want to force the install of GPU drivers if there's no GPU?  Won't that break the instance?

Reply:
Fair point but imagine this. Now we have only CPU and Nvidia GPU. What if we decide we want to support AMD GPUs, what about Intel GPUs? On the other hand we might never need to touch this flag ever. In the past however we have seen that boolean configs are cumbersome enspecialy during upgrades, my suggestion is to use a string. On your question, you may want to install GPU drivers in case you will add the GPU later on the host, this is the case for google cloud instances.

Reply:
Ah okay, so upgrading the config from boolean to string is a pain?  I didn't know that.  I get your point on Nvidia vs AMD vs Intel vs nothing so I'll change the config option to reflect this.